sepal_ui/__init__.py

```diff
@@ -1,56 +1,11 @@
-from types import SimpleNamespace
 from pathlib import Path
-from configparser import ConfigParser
+from sepal_ui.frontend.styles import color, get_theme, theme
 
 import ipyvuetify as v
 
-from sepal_ui.frontend import styles
-
 __author__ = """Pierrick Rambaud"""
 __email__ = "pierrick.rambaud49@gmail.com"
 __version__ = "2.7.0"
 
```

Comment:
you confirm that it's still possible to call color as such: 

```
import sepal_ui 

sepal_ui.color.primary
>>> the selected color hex code
```

I'm in an airport it's complex to run sepal

Reply:
Yes, that will work since we are importing `color` and the other required functions (I've tested it).